src/app.js

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios';
+
+async function getLand(land) {
+    try {
+        const uri = "https://restcountries.com/v2/name/" + land;
+        const result = await axios.get(uri);
+        console.log(result.data[0]);
+        const {name, subregion, population, flag, capital, currencies} = result.data[0];
+
+
+
+        let currenciesText = "";
+        if (currencies.length > 1){
+            for (let i = 0; i < currencies.length; i++) {
+                if(i === currencies.length - 1){
+                    currenciesText += currencies[i].name + " currencies";
+                }
+                else{
+                    currenciesText += currencies[i].name + ", ";
+                }
+            }
```

Comment:
Goed gebruik gemaakt van de for-loop. Wellicht dat je als vervolgstap iets met de [.join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join) kan doen

Reply:
Kun je uitleggen hoe? 'currencies' is an array of objects...if I place currencies.join(','), het zou objecten weergeven, geen namen van deze objecten...
![image](https://user-images.githubusercontent.com/105750397/189107027-834c9918-a106-4d92-a98a-c4fd47735b85.png)
![image](https://user-images.githubusercontent.com/105750397/189107083-5835e0e0-1101-40ef-948c-081d9a5b52d5.png)